### DISCUSSION TITLE
clear chat

### DISCUSSION THREAD

**Discussion by elaineandsparky** [EssentialsX/Essentials#4631] [Ideas and Feature Suggestions] [answered]
I know /clearchat used to be a thing but unless I'm doing something wrong it's not working any more or wasn't with this plugin.

Any chance we can get it back? so if someone says something bad staff can just do /clearchat to get rid of it

**Comment by pop4959** [comment, marked as answer]
You can clear chat for yourself in Minecraft using the F3-D hotkey. A global clear chat command has never been a feature in Essentials and is very unlikely to be added unless better support is added for it in the game, as currently the only way to do so is by flooding the chat with blank messages (there are other plugins that do this that you can use if you really want to go that route).

**Reply by elaineandsparky** [reply]
Thank you for quick reply. Must have been a different plugin that was used on old server.
